core/ir/decode_shared.c

```diff
@@ -153,6 +153,7 @@ const char *const size_names[] = {
     "OPSZ_8x16",
```

Comment:
Maybe add to the changelist https://github.com/DynamoRIO/dynamorio/blob/master/api/docs/release.dox#L138?